### DISCUSSION TITLE
How to enable reference action?

### DISCUSSION THREAD

**Discussion by a-wf** [prisma/prisma#9482] [Q&A] [answered]
Hi

I have a project that use Prisma for months, and i notice that the cascade deletion feature is released for version 2.26.0 and later.

So I tried to set it for a new table schema with v2.30.3 of Prisma, but during the client generation, I get:

```
Error: Schema parsing
error: Error parsing attribute "@relation": The relation field `xxx` on Model `xxxx` must not specify the `onDelete` argument in the @relation attribute without enabling the `referentialActions` preview feature.
```
And when I added the `referentialActions` in preview feature array, I have some warning about `Preview feature "referentialActions" is deprecated. The functionality can be used without specifying it as a preview feature.`

Then what should I do? Because with the `preview feature` tag, the client generation doesn't fail with error. But I get no change on `git diff` between old client and new client, neither for the migration folder

**Comment by ryands17** [comment, marked as answer]
@akdj 👋 

It seems that you're not using the latest version of Prisma. I would suggest updating `prisma` and `@prisma/client` and checking again.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
